LDKNodeMonday/View Model/Home/OnboardingViewModel.swift

```diff
@@ -8,23 +8,23 @@
 import LDKNode
 import SwiftUI
 
-class OnboardingViewModel: ObservableObject {
-    @AppStorage("isOnboarding") var isOnboarding: Bool?
-    @Published var networkColor = Color.gray
-    @Published var onboardingViewError: MondayError?
-    @Published var seedPhrase: String = "" {
+@Observable
+class OnboardingViewModel {
+    var networkColor = Color.gray
+    var onboardingViewError: MondayError?
+    var seedPhrase: String = "" {
         didSet {
             updateSeedPhraseArray()
         }
     }
-    @Published var seedPhraseArray: [String] = []
-    @Published var selectedNetwork: Network = .signet {
+    var seedPhraseArray: [String] = []
+    var selectedNetwork: Network = .signet {
         didSet {
             do {
                 let networkString = selectedNetwork.description
                 try KeyClient.live.saveNetwork(networkString)
-                selectedURL = availableURLs.first ?? ""
-                try KeyClient.live.saveEsploraURL(selectedURL)
+                self.selectedEsploraServer = availableEsploraServers.first!
```

Comment:
change requested: lets just use `??` here instead of `!`, even though maybe later we will want to handle this even better by throwing/showing an error to the user in the best way

```suggestion
                self.selectedEsploraServer = availableEsploraServers.first ?? EsploraServer(name: "", url: "")
```